packages/client/components/forms/post-job/1_JobBasics.tsx

```diff
@@ -24,10 +24,10 @@ type JobBasicProps = {
   goBack: () => void;
   onChange: (e: React.FormEvent) => void;
   formData: {
-    jobtitle: string;
-    jobdescription: string;
-    jobposition: string;
-    jobtype: string;
+    title: string;
+    description: string;
+    position: string;
+    type: string;
```

Comment:
Just a trivial comment and no changes required.

But you can use TypeScript's utility "Pick", e.g.
```ts
formData: Pick<definitions["jobs"], 'title' | 'description' | 'position' | 'type'>
```